pyproject.toml

```diff
@@ -22,6 +22,20 @@ black = {version = "^21.5b1", allow-prereleases = true}
 types-requests = "^2.25.6"
 requests-cache = "^0.8.1"
 
+[[tool.mypy.overrides]]
+module = [
+    "backoff",
+]
+ignore_missing_imports = true
```

Comment:
just curious, why is this needed?

Reply:
@ericboucher when a package like `backoff` is missing library stubs (such as `types-requests` for `requests`) or a `py.typed` file, mypy cannot get typing information from it.

This is the same as putting `# type: ignore` in front of any `import backoff` lines to make mypy happy.

Reply:
So written like this, it only gets applied to the "backoff" module?